ocaml/idl/datamodel_cluster.ml

```diff
@@ -194,6 +194,9 @@ let t =
              (Some (VString Constants.default_smapiv3_cluster_stack))
            "Simply the string 'corosync'. No other cluster stacks are \
             currently supported"
+       ; field ~qualifier:StaticRO ~lifecycle ~ty:Int "cluster_stack_version"
+           ~default_value:(Some (VInt 2L))
```

Comment:
Would it be better to following the way of the "cluster_stack" field, making the default value from `Constants.default_smapi_v3_cluster_stack_version`?

Reply:
Hopefully the version will never be something like `2.5`.

Reply:
sounds good, avoid magic constants

Reply:
This can be counter-intuitive, I will add a comment in the doc.

Reply:
>We will always create a corosync3 cluster when the user does not specify anything

But only once we are ready for Corosync 3, i.e. not yet.